docs/sop.rst

```diff
@@ -0,0 +1,135 @@
+Operating Clowder
+=================
+
+Two primary aspects of operating Clowder: Operating the apps managed by Clowder and operating
+Clowder itself.
+
+Contrary to many other deployment models where apps configure themselves, an application that is
+managed by Clowder will have configuration presented to it and is expected to use it. Clowder
+governs many different aspects of an application's configuration, from defining the port it should
+listen on, to metrics, kafka topics, service hostnames, and more. In this way a common configuration
+format is presented to the application no matter the envrionment it is running in, enabling a far
+easier development experience.
```

Comment:
ok yeah you get to the benefits here, but I wonder if this should be explained at the beginning of the paragraph rather than the end?